guides/doc-Quickstart/master.adoc

```diff
@@ -19,15 +19,15 @@ For a full installation guide, see {InstallingServerDocURL}[_{InstallingServerDo
 ifdef::foremanctl[]
 The full installation guide is currently provided only for the Puppet-based installation.
 
-:FeatureName: The `foremanctl` installer
+:FeatureName: The `foremanctl` deployment tool
```

Comment:
In the other file you call it "utility", not "tool", while both refer to the same thing, I guess it's easier if the same word is used?

ACK either way.

Reply:
We should add a description in the About section of https://github.com/theforeman/foremanctl that we can reuse here. For now it can only deploy, but we've also been talking about merging foreman-maintain functionality into it so we may need to revisit this if/when we do.